<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkXnatSimpleSearchViewControls</class>
 <widget class="QWidget" name="QmitkXnatSimpleSearchViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>975</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>xnat_icon.ico</normaloff>xnat_icon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="labelWarning">
       <property name="styleSheet">
        <string notr="true">QLabel { color: rgb(255, 0, 0) }</string>
       </property>
       <property name="text">
        <string>Please insert your XNAT host address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inHostAddress">
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Examples:&lt;/p&gt;&lt;p&gt;&amp;quot;http://localhost:8080/xnat&amp;quot;&lt;/p&gt;&lt;p&gt;&amp;quot;http://central.xnat.org:80&amp;quot;&lt;/p&gt;&lt;p&gt;&amp;quot;https://xnat.myserver.de:443&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>http://localhost:12345/xnat</string>
       </property>
       <property name="placeholderText">
        <string>http(s)://host:port/pathToXnatServerInstance</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="inUser">
         <property name="text">
          <string>admin</string>
         </property>
         <property name="placeholderText">
          <string>User</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inPassword">
         <property name="minimumSize">
          <size>
           <width>53</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>admin</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QComboBox" name="objectComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Project</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Subject</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Experiment</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="objectLabel">
         <property name="text">
          <string>Matching Object</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="termLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="termLabel">
         <property name="text">
          <string>Search Term</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="buttonStartSearch">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Get all projects</string>
       </property>
       <property name="text">
        <string>Start Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="treeView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
